apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.Tests/Utils.cs

```diff
@@ -43,16 +43,10 @@ public static string GetTestId()
             return Guid.NewGuid().ToString();
         }
 
-        /// <summary>A sizer for ints.</summary>
-        public class IntSizer : ISizer<int>
+        public static class IntSizer
```

Comment:
Possibly `ConstantSizer`? (That could be generic too, if you ever want to get a constant size for other types.)